src/platforms/esp32/components/avm_builtins/network_driver.c

```diff
@@ -404,10 +410,15 @@ static wifi_config_t *get_sta_wifi_config(term sta_config, GlobalContext *global
     return wifi_config;
 }
 
-static char *get_default_device_name()
+static char *get_default_device_name(esp_mac_type_t Interface)
```

Comment:
let's change it to lowercase: `esp_mac_type_t Interface` -> `esp_mac_type_t interface`.